doc/src/dev/tools.rst

```diff
@@ -0,0 +1,65 @@
+Tools
+=======
+
+rocTX
+------------
+MIGraphX driver provides `roctx` knob which can be used with `rocprof` binary to get marker timing information for each MIGraphX operator.  
+In order to help user to process timing information, rocTX helper script is provided at `/tools/roctx.py`.
+The `roctx.py` helper script provides two main functionality: `run` and `parse`. Available knobs and usage are given below:
+
+::
+
+    Usage: roctx.py [-h] [--json-path json_path] [--out out]
+    [--study-name study-name] [--repeat repeat] [--parse]
+    [--run run] [--debug]
+
+.. option::  --run
+
+Runs MIGraphX with rocTX command and given `migraphx-driver` knobs, and then parses the results, providing GPU kernel timing information.
```

Comment:
Maybe say "run `migraphx-driver roctx` command". Also does this run `migraphx-driver` on the system path or is it a specific path?